app/locale/lv.coffee

```diff
@@ -1,44 +1,44 @@
 module.exports = nativeDescription: "latviešu", englishDescription: "Latvian", translation:
 
 #  new_home:
```

Comment:
u need to remove `#` from this line to activate its children keys. So, in this case, to activate `title` and `meta_keywords`, u need to enable its parent `new_home`